pyfr/solvers/acnavstokes/elements.py

```diff
@@ -6,6 +6,10 @@
 
 class ACNavierStokesElements(BaseACFluidElements,
                              BaseAdvectionDiffusionElements):
+    @staticmethod
+    def grad_con_to_pri(cons, grad_cons, cfg):
+        return [grad for grad in grad_cons]
```

Comment:
Can we just return grad_cons or list(grad_cons) here?

Reply:
Done